includes/class-wpfaevent-deactivator.php

```diff
@@ -6,31 +6,14 @@
  * @link       https://fossasia.org
  * @since      1.0.0
  *
- * @package    Wpfaevent
- * @subpackage Wpfaevent/includes
+ * @package    WPFA_Event
+ * @subpackage WPFA_Event/includes
  */
 
-/**
- * Fired during plugin deactivation.
- *
- * This class defines all code necessary to run during the plugin's deactivation.
- *
- * @since      1.0.0
- * @package    Wpfaevent
- * @subpackage Wpfaevent/includes
- * @author     FOSSASIA <contact@fossasia.org>
- */
-class Wpfaevent_Deactivator {
+class WPFA_Event_Deactivator {
 
-	/**
-	 * Short Description. (use period)
-	 *
-	 * Long Description.
-	 *
-	 * @since    1.0.0
-	 */
```

Comment:
Why do these comments need to be deleted? This is not executed, but it could help in future development. So, what is the goal?